### PR TITLE
Bump script

### PR DESCRIPTION
**Pull request by MarinPostma**
add a script that replace the current version with a bumped version in all workspaces

### CODE REVIEW

File: bump.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+then
+	printf "error: a version number must be provided\n"
+	exit 1
+fi
+
+# find current version
+CURRENT_VERSION=$(cat **/*.toml | grep meilisearch | grep version | sed 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/' | sed "1q;d")
+
+# bump all version in .toml
+printf "bumping from version $CURRENT_VERSION  to version $NEW_VERSION\n"
```
Comment (Kerollmops) [inline bump.sh:15]:
```suggestion
echo "bumping from version $CURRENT_VERSION to version $NEW_VERSION"
```

File: bump.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+while true
+do
+	read -r -p "Continue (y/n)?" choice
+	case "$choice" in
+		y|Y ) break;;
+		n|N ) printf "aborting bump\n" && exit 0;;
+		* ) printf "invalid choice\n";;
```
Comment (Kerollmops) [inline bump.sh:22]:
```suggestion
		n|N ) echo "aborting bump" && exit 0;;
		* ) echo "invalid choice";;
```

Comment (Kerollmops) [reply bump.sh:22]: Are those double semi-colon `;;` mandatory?

Comment (MarinPostma) [reply bump.sh:22]: yes they are

File: bump.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+then
+	echo "error: a version number must be provided"
+	exit 1
+fi
+
+# find current version
+CURRENT_VERSION=$(cat **/*.toml | grep meilisearch | grep version | sed 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/' | sed "1q;d")
+
+# bump all version in .toml
+echo "bumping from version $CURRENT_VERSION  to version $NEW_VERSION"
```
Comment (Kerollmops) [inline bump.sh:15]:
```suggestion
echo "bumping from version $CURRENT_VERSION to version $NEW_VERSION"
```

Comment (Kerollmops) [inline bump.sh:15]:
```suggestion
echo "bumping from version $CURRENT_VERSION to version $NEW_VERSION"
```

**Pull request merged**